src/commands/zork.js

```diff
@@ -0,0 +1,44 @@
+const BaseCommand = require('../base-commands/base-command');
+
+class Zork extends BaseCommand {
+  constructor(argv, config, secureStorage) {
+    super(argv, config, secureStorage);
+    this.exec = require('await-exec');
+    this.findZork = () => require('zorkjs'); // eslint-disable-line node/no-extraneous-require,node/no-missing-require
+  }
+
+  async run() {
+    await super.run();
+
+    let launchZork = null;
+    try {
```

Comment:
I'm not entirely sure I follow this code flow. My hunch is a lot of this code is a byproduct of not yet implementing the catch component.

If it fails to load the module, then in that catch at line 16 I would probably move lines 21-28

Because as it stands there should be no reason why line 20 `!launchZork` should ever reasonably not be true after this try/catch completes. If the subsequent catch at line 26 is hit, we should be throwing an error at that point and not letting code execution to continue